supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala

```diff
@@ -32,6 +33,14 @@ final case class TesTask(jobDescriptor: BackendJobDescriptor,
   private val workflowDescriptor = jobDescriptor.workflowDescriptor
   private val workflowName = workflowDescriptor.callable.name
   private val fullyQualifiedTaskName = jobDescriptor.taskCall.fullyQualifiedName
+  private val workflowExecutionIdentityConfig: Option[String] =
+    configurationDescriptor.backendConfig
+      .getAs[String]("workflow_execution_identity")
+  private val workflowExecutionIdentifyRuntime: Option[String] =
```

Comment:
Nitpick, I think this is more properly named `workflowExecutionIdentityOption` since it comes from workflow options, not the WDL task runtime.